themes/adminimal_medela/source/js/publication-description.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @file
+ * Custom functionality for publication rules.
+ */
+(function ($, Drupal, window) {
+
+  'use strict';
+
+  Drupal.behaviors.adminimalMedelaPublicationDescription = {
+    attach: function (context, settings) {
+
+      //Only show description field if a file is uploaded.
+      $(".page-node-type-h-publication .vertical-tabs__menu-item").on("click", function () {
```

Comment:
The performance of this jQuery code could be improved. There are several line which initialize a jQuery object starting with ".page-node-type-h-publication". It would be more efficient to initialise the ".page-node-type-h-publication" object once and then use that as a base for other selectors. This means jQuery doesn't have to search the whole DOM each time. Instead it one has to search withing the ".page-node-type-h-publication" object.

```
var $pageNodeTypeHPublication = $(".page-node-type-h-publication", context);
...
$(".paragraph-type--para-h-resource-part .item-container", $pageNodeTypeHPublication).each( ... );
...
...
var $fieldDescription = $(".field--name-field-h-description", $pageNodeTypeHPublication);
...
...
```